src/Grafana/dashboards/aspnetcore.json

```diff
@@ -413,6 +413,7 @@
           "showLegend": true
         },
         "tooltip": {
+          "maxHeight": 600,
```

Comment:
This is probably junk and I can remove that - stuff Grafana adds when doing a JSON export